config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -0,0 +1,26 @@
+presubmits:
+  kubernetes-sigs/image-builder:
+  - name: pull-azure-ubuntu-1804
+    labels:
+      preset-azure-cred: "true"
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
```

Comment:
1h should be plenty but might adjust later especially if we start doing more stuff like run node conformance down the line